### ISSUE TITLE
Communication score bug

### ISSUE THREAD

**Issue by VALVEFIXTHEBUG** [ValveSoftware/Dota2-Gameplay#13896]
### Description

Мой уровень вежливости не растет, несмотря на большое количество положительных отзывов и малое количество репортов. На скриншотах можете в этом убедиться. 
My level of communication score is not increasing despite a lot of positive feedback and few reports. You can see for yourself on the screenshots. 
Мой уровень порядочности меняется каждые 15 игр, чего не скажешь о вежливости. За последние 30 матчей он никак не изменился, а остался прежним (5745). 
My level of behavior score changes every 15 games, which is not the case with communication score. It hasn't changed in any way over the last 30 games, but has stayed the same (5745). 
С данной проблемой, как мне удалось убедиться, сталкивается немалое количество игроков. Полагаю, что ее необходимо решить первостепенно, ибо вы заявляли сообществу о том, что сконцентрируетесь на игровом процессе. 
This is a problem that I have seen a lot of players have to deal with. I think it should be addressed as a priority, as you have stated to the community that you will focus on gameplay. 

### Screenshots

![1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/152329700/2c054d31-c821-4231-8899-6518ae5d53e0)

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/152329700/0f19ba74-37a7-4a81-9762-71607296899a)

**Comment by legacy1234** [comment]
100 more posts and maybe they will finally adress this issue.

**Comment by Questy33** [comment]
Has your MMR dropped significantly in 2 months too?

**Comment by VALVEFIXTHEBUG** [comment]
> Has your MMR dropped significantly in 2 months too?

no. i gained 500 mmr (from 6200 to 6700) in 1 month, then i left the game, my score significantly dropped from 9000 to 7500 (both communication and behavior) so i couldnt use the voice chat. probably i got reported for pinging too much, because its actually impossible for me to convey the ideas to people without voice. then i got reported several times (my communication was the same as at 9000 score, the number that had been stable for 2 months since the beginning of september) so now i cant communicate at all. i tried to gain some points by playing 30 turbo games, but its useless. btw my behavior is growing as you can see, so i dont ruin games i just cant call people for actions and now i have 6400 mmr, ONLY because i cant communicate.

**Comment by VALVEFIXTHEBUG** [comment]
i guess it will be better if players with low communication score would be pre-muted (like before), but not be limited access to communicate at all. so someone who wants could unmute the low communication score player.

**Comment by legacy1234** [comment]
It is a bug, the score is stuck at same value, that is not supposed to happen. It should still increase/decrease. @VALVEFIXTHEBUG

**Comment by VALVEFIXTHEBUG** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/152329700/d65e7517-74d9-486d-b55b-c368a78c7944)
Another one
